appveyor.yml

```diff
@@ -42,15 +40,9 @@ clone_depth: 1
 shallow_clone: false
```

Comment:
Can be true, I guess. Saves bandwidth

Reply:
I did not even look at this section.  Don't know what it is for.
I also don't know if we need to do anything with the wheels.

Reply:
let's not bother with wheels. 
I know you did not change shallow_clone, just seeing it here. Shallow means you just download git history to depth of 1 (or a small number), not the whole history of the project. As that is not needed for the build